.github/workflows/build_wheels_linux.yml

```diff
@@ -134,6 +162,9 @@ jobs:
         run: |
           set -euxo pipefail
           source "${BUILD_ENV_FILE}"
+          if [[ ${{ inputs.architecture }} == 'ARM64' ]]; then
+            ${CONDA_RUN} conda install -y libpng jpeg
```

Comment:
Should we add these in https://github.com/pytorch/vision/blob/main/packaging/pre_build_script.sh (vision pre-script) instead?

Reply:
Sounds good, let me do this in subsequent PR in torchvision